accepted/0017-add-funding-support.md

```diff
@@ -0,0 +1,117 @@
+# Add `funding` support to `package.json`
+
+## Summary
+
+This RFC identifies an initial means of tooling to describe & notify consumers of a package's monetary support information.
+
+## Motivation
+
+Package maintainers want to clearly indentify how their software is currently, or could be in the future, supported monetarily.
+
+## Detailed Explanation
+
+* See [Prior Art](#prior-art)
+* Provide a means to **reference** existing backers or funding opportunities in `package.json`
+* Provide a means to **define the type** of backing or funding opportunities in `package.json`
+* Provide a means to **notify** package consumers of backers or funding opportunites with the cli
+* Provide a means to **view** dependency's backers or funding opportunites with the cli
+* Provide a means to **view** a package's backers or funding opportunites
+
+##  Rationale & Alternatives
+* Rationale: This is a straightforward and easily implemented solution to take the first step toward supporting Open Source Package Maintainence sustainability
+* Alternative: Do nothing
+* Alternative: Continue debating a larger, more complex schema /w correspondingly more complex tooling
+
+## Implementation
+
+* Add a `funding` field to `package.json`
+  * supports a string URL or object with...
+  * keys that support arbitrary string identifiers
+    * ex. `"sponsor"`, `"sponsors"`, `"donations"`, `"bounties"`, `"contributors"`, `"patrons"` etc.
+  * values that represent a string URL or an array of URLs
+* Add notification at the end of output of package installation that references the number of packages with `funding` defined 
+  * ex. `23 packages are looking for funding. Run "npm fund" to find out more.`
+* Add `--no-fund` flag to opt-out of the funding notification when installing
+* Add `npm fund` subcommand which will: open the URL (string) **OR** list the funding references defined (object)
```

Comment:
The open URL scenario seems like something you'd do if an application has only _one_ fund-seeking dependency which has provided only a _single_ URL as its funding option. This seems like it will be a pretty rare case, especially as more packages start to incorporate the `funding` attribute.

Why bother opening the URL at all? It will happen sporadically at best.

Reply:
@djsauble this is a good point and I will update the RFC based on that note. This was intended to follow a similar pattern to the `npm repo <pkg>` command which tries to open the correct URL.